### ISSUE TITLE
See if historical essay content can be pulled into title page

### ISSUE THREAD

**Issue by sseymore** [uoregon-libraries/oregon-oni#30]
like http://nebnewspapers.unl.edu/lccn/46032385/

**Comment by jechols** [comment]
Yes, but there isn't a terribly easy way to do it.  It sounds like NE has been doing this by just having somebody put the HTML directly into the database.

If we are interested in this, I think we should brainstorm some, and probably with others on the ONI team.  Otherwise it'll be a pain to add and change essays, and it'll always require a developer's involvement.

**Comment by jechols** [comment]
I've cleared the milestone.  I *think* this is something to discuss with the community, as opposed to trying to get implemented ASAP.  @sseymore feel free to correct me, though - if this is important, we can always script something up to pull from the blog as a way to bridge the gap temporarily.

**Comment by jechols** [comment]
Removing the milestone; we've decided to wait a little longer to figure out if it's worth a community-level fix for essays

**Comment by sseymore** [comment]
Cool, thanks, Jeremy!
